scripts/vaccination_alerts/vaccination-alert-template.html

```diff
@@ -0,0 +1,111 @@
+          </div>
+        </a>
+      </div>    
+      {{/if}}
+
+      <div class="section description center-text" style="color: #4f4f4f; margin-top: 24px; text-align: center;">
+        Because eligibility varies throughout {{locationName}}, you may also want to check your county or city’s health department website.
+      </div>
+      
+    </div>
+
+    <div class="footer" style="background-color: #f2f2f2; margin-top: 50px; padding: 50px 24px; text-align: center;">
+      <div class="centered-column" style="margin: 0 auto; max-width: 400px;">
+
+        <div class="footer__logo" style="margin-bottom: 24px;">
+          <a href="https://covidactnow.org" style="line-height:inherit" target="_blank">
+            <img src="https://data.covidactnow.org/thermometer_screenshot/canlogo.png" alt="Covid Act Now Logo" width="160"/>
+          </a>
+        </div>
+
+        <div class="footer__section" style="margin-bottom: 50px;">
+          <p>
+          Covid Act Now is an independent 501(c)(3) nonprofit founded by volunteers in March 2020. We help people make informed decisions by providing timely and accurate data about COVID in the U.S.
+          </p>
+          <a class="footer__link" style="color: #4f4f4f;" href="https://covidactnow.org/about">Learn more about us</a>
+        </div>
+
+        <div class="footer__section" style="margin-bottom: 50px;">
+          <h2 style="color: #000; font-size: 18px; margin-bottom: 5px; text-align: center;">Donate</h2>
+          <p style="margin: 5px auto;">Find these alerts helpful?</p>
+          <a class="footer__link" style="color: #4f4f4f;" href="https://covidactnow.org/donate">Support us with a donation</a>
+        </div>
+
+        <div class="footer__section" style="margin-bottom: 50px;">
+          <h2 style="color: #000; font-size: 18px; margin-bottom: 5px; text-align: center;">Feedback</h2>
+          <p style="margin: 5px auto;">
+            Tell us how we can make our alerts more useful for you, at <a class="footer__link" style="color: #4f4f4f;" href="mailto:info@covidactnow.org">info@covidactnow.org</a>
+          </p>
+        </div>
+
+        <div class="footer__section" style="margin-bottom: 50px;">
+          <h2 style="color: #000; font-size: 18px; margin-bottom: 5px; text-align: center;">Help others get alerts</h2>
+          <p style="margin: 5px auto;">Anyone can subscribe using this link:</p>
+          <a class="footer__link" style="color: #4f4f4f;" href="https://covidactnow.org/alert_signup">
+            https://covidactnow.org/alert_signup
+          </a>
+        </div>
+
+        <div class="footer__section" style="margin-bottom: 50px;">
+          <h2 style="color: #000; font-size: 18px; margin-bottom: 5px; text-align: center;">Change your alert settings</h2>
+          <p style="margin: 5px auto;">
+            <a class="footer__link" style="color: #4f4f4f;" href="https://covidactnow.org/alert_signup">Edit your locations</a>
```

Comment:
I think this should go to the /alert_unsubscribe page (same as the actual "unsubscribe from all alerts" link below).  And it needs to encode the email address into the link. https://github.com/covid-projections/covid-projections/blob/11c10502451ef5ee08fc1256875e8958c9dd0529/scripts/alert_emails/utils.ts#L56